### ISSUE TITLE
Cannot open .bib from Google Drive folder using ubuntu

### ISSUE THREAD

**Issue by joangog** [JabRef/jabref#10640]
### JabRef version

Latest development branch build (please note build date below)

### Operating system

Other (please describe below)

### Details on version and operating system

Ubuntu 22.04.1 LTS

### Checked with the latest development build (copy version output from About dialog)

- [X] I made a backup of my libraries before testing the latest development version.
- [X] I have tested the latest development version and the problem persists

### Steps to reproduce the behaviour

1. Install:
JabRef 5.12--2023-11-13--a7fbf85
Linux 6.2.0-33-generic amd64 
Java 21.0.1 
JavaFX 21.0.1+6

2. Select "Open Library"
3. Google drive folder is missing. 

or

2. Right click on .bib file inside Google Drive folder.
3. Select "Open with JabRef"
4. JabRef opens but the library is empty and shown as "untitled.bib"

### Appendix

...
<details>

<summary>Log File</summary>

```
Paste an excerpt of your log file here
```

</details>

**Comment by Siedlerchr** [comment]
Hi,
do you see your google account on the other locations e.g. as the user described here? 
https://discourse.jabref.org/t/cannot-open-save-bib-file-on-google-drive-from-linux/3879/6?u=siedlerchr

Otherwise, you probably need to mount your gdrive folder somehow

**Comment by joangog** [comment]
> Hi, do you see your google account on the other locations e.g. as the user described here? https://discourse.jabref.org/t/cannot-open-save-bib-file-on-google-drive-from-linux/3879/6?u=siedlerchr
> 
> Otherwise, you probably need to mount your gdrive folder somehow https://vitux.com/how-to-access-your-google-drive-account-in-ubuntu/

Yep it's mounted! I saw that answer, I tried updating but it didn't work for me unfortunately...
